subprojects/gstreamer/plugins/tracers/gstframerate.c

```diff
@@ -186,13 +417,59 @@ do_push_buffer_pre (GstTracer *tracer, GstClockTime ts, GstPad *pad,
     gst_object_unref (peer_pad);
 }
 
+static void
+gst_framerate_tracer_constructed (GObject *object)
+{
+  GstFramerateTracer *self = GST_FRAMERATE_TRACER (object);
+  gchar *params, *tmp;
+  GstStructure *params_struct = NULL;
```

Comment:
please add
```c
G_OBJECT_CLASS (parent_class)->constructed (object);
```
not chaining up is wrong even though there won't be a visible error _for now_